2-mandatory/8-password-validator.js

```diff
@@ -23,7 +23,15 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+function validatePasswords(passwords) {
+  return passwords.map((aString,index) => aString.length >= 5 && 
+                                        containsUppercaseLetter(aString) &&
+                                        containsLowercaseLetter(aString) &&
+                                        containsNumber(aString) &&
+                                        containsSymbol(aString) &&
+                                        !(passwords.slice(0,index).includes(aString)))
```

Comment:
What a great solution to check if the password is repeated! I struggled so much with this bit (lol thank goodness for stack overflow), but I like how you used slice to check! very cool